README.md

```diff
@@ -1,28 +1,26 @@
 # Blockchain Developer Path
 
-Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of web3community. We gather content about Blockchain developers, you don't need to watch it all but choose what content is best for you.
+Here you will learn how to become one of the Dapp Developers and collaborate with other people inside of Web3 Community. We gather content about Blockchain developers. You don't need to see all the resources, but choose what content is best for you.
```

Comment:
I think this one needs no change. By web3community, we are mentioning our community. Not the general Web3 Community.

Reply:
+1

Reply:
web3community is the name of our community. And Web3, call it a tech stack or a new technology. Community focusing on Web3 is called a Web3 Community. We are also a Web3 Community, just our name is web3community. I know it's a bit confusing for you. Maybe @vinzvinci can explain in more simple words.

Reply:
Updated now. :)